doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst

```diff
@@ -0,0 +1,167 @@
+.. _ncs_release_notes_2699_cs1:
+
+|NCS| v2.6.99-cs2 Release Notes
+###############################
+
+.. contents::
+   :local:
+   :depth: 3
+
+|NCS| v2.6.99-cs2 is a **customer sampling** release, tailored exclusively for participants in the nRF54H20 customer sampling program.
+Do not use this release of |NCS| if you are not participating in the program.
+
+The release is not part of the regular release cycle and is specifically designed to support early adopters working with the nRF54H20 device.
+It is intended to provide early access to the latest developments for nRF54H20, enabling participants to experiment with new features and provide feedback.
+
+The scope of this release is intentionally narrow, concentrating solely on the nRF54H20 device.
+You can use the latest stable release of |NCS| to work with other boards.
+
+All functionality related to nRF54H20 is considered experimental.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Peripherals support
+===================
+
+* Added experimental support for the following peripherals on the nRF54H20 PDK:
+
+  * ADC
+  * CLOCK
+  * GPIO
+  * GRTC
+  * I2C
+  * MRAM
+  * PWM
+  * RADIO (Bluetooth Low Energy and 802.15.4)
+  * SPI
+  * UART
+  * USB
```

Comment:
is USB supported?

Reply:
@carlescufi @kapi-no Any insight here?

Reply:
@gmarull @carlescufi @kapi-no @pdunaj @miha-nordic Any insight here?

Reply:
not supported yet